cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -0,0 +1,408 @@
+import unittest
+import pandas as pd
+from cellxgene_schema import ontology
+from cellxgene_schema import validate
+import numpy
+import fixtures.examples_validate as examples
+from scipy import sparse
+
+
+class TestFieldValidation(unittest.TestCase):
```

Comment:
What does `Field` mean in this case? These tests are not testing validation of  specific AnnData object, but rather the schema file itself, and some unit tests for `_validate_curie`.  Is this is cohesive grouping of tests? Maybe the class be renamed for clarity.